docs/tutorials/enterprise_beginner/3_using_existing_blocks.md

```diff
@@ -108,15 +94,25 @@ Finally, we should add some styling to the block. Add the following CSS to the e
 }
 ```
 
-If you refresh the front page now, you should see the new look of the Content List block.
+At this point you can start adding blocks to the Landing Page. This is done in the StudioUI Edit mode (Page tab) by simply dragging the required block from the menu on the right to the correct zone on the page.
+
+![Content List block dragged onto a page](img/enterprise_tut_drag_block.gif)
+
+Let's start with the simplest one. Drag a Content List block from the menu to the right column. Click the (still empty) block and enter its settings. Here you name the block and decide what it will display. Choose the Dog Breed Catalog folder as the Parent, select Dog Breed as the Content Type to be displayed, and choose a limit. In this case you'll display the first three Dog Breeds from the database.
```

Comment:
Do we use the full form of `will`?

Reply:
Not sure, it was in the original so I left it as we do not use formal language.